### PR TITLE
Minor buff to Chem Master 

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
* Chem Master now holds 200 units

## Why It's Good For The Game
Since a lot of chems are nerfed, making some advanced chem is painful in this state.
We'll likely merge the sleeper PR, but adjusting chem ratio is difficult. I am going to make this easy to produce,

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: ChemMaster now says which amount chem units it can hold.
balance: ChemMaster now can hold 100 units more regardless of beakers inside (roundstarts with 200 max units)
/:cl:

### CODE REVIEW

**Comment by Isy232** [comment]
> Chem Master now holds 200 units

Oh, great, you have made it so chemists no longer have to upgrade their chem masters as much by having them start with large beakers instead of small beakers...

Oh.... oh no.... they just take 4 beakers to make now...

I'm not against it, but would it not be easier to keep the current 2 beaker chem masters but have the ones that spawn at roundstart include large beakers instead of small ones?

I dont know machine code well enough, if this is the best way then go ahead.

**Comment by EvilDragonfiend** [comment]
> > Chem Master now holds 200 units
> 
> Oh, great, you have made it so chemists no longer have to upgrade their chem masters as much by having them start with large beakers instead of small beakers...
> 
> Oh.... oh no.... they just take 4 beakers to make now...
> 
> I'm not against it, but would it not be easier to keep the current 2 beaker chem masters but have the ones that spawn at roundstart include large beakers instead of small ones?
> 
> I dont know machine code well enough, if this is the best way then go ahead.

hmm, maybe I should just increase it by a fixated value regardless of beaker level. machine code for beakers are specifically bullshit.

**Comment by EvilDragonfiend** [comment]
Okay, I changed it to 2 large beakers, and I checked it's working well locally.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Since you are saying this is a buff, I want to point out that this will make saline glucose 3x less effective, as it now metabolizes 3x faster but performs its effect at the exact same rate. This means that you need 90u of saline to do the same thing as 30u before. The title and body of the PR seem to portray this as a buff.

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 1]
```diff
@@ -47,7 +47,7 @@
 /datum/chemical_reaction/salglu_solution
 	name = "Saline-Glucose Solution"
 	id = /datum/reagent/medicine/salglu_solution
-	results = list(/datum/reagent/medicine/salglu_solution = 3)
+	results = list(/datum/reagent/medicine/salglu_solution = 9)
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: This doesn't make any sense.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
I don't get the point of making saline 3x bigger in everything

**Comment by EvilDragonfiend** [comment]
the PR is now limited to chemmaster. The idea about saline seems to be done after some chem changes...

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**